packages/auth-sveltekit/src/server.ts

```diff
@@ -290,6 +299,104 @@ export class ServerRequestAuth extends ClientAuth {
     return { tokenData };
   }
 
+  async magicLinkSignUp(data: { email: string } | FormData): Promise<void> {
+    if (!this.config.magicLinkFailurePath) {
+      throw new ConfigurationError(
+        `'magicLinkFailurePath' option not configured`
+      );
+    }
+    const [email] = extractParams(data, ["email"], "email missing");
+
+    const { verifier } = await (
+      await this.core
+    ).signupWithMagicLink(
+      email,
+      `${this.config.authRoute}/magiclink/callback?isSignUp=true`,
+      new URL(this.config.magicLinkFailurePath, this.config.baseUrl).toString()
+    );
+
+    this.cookies.set(this.config.pkceVerifierCookieName, verifier, {
+      httpOnly: true,
+      sameSite: "strict",
+      path: "/",
+    });
```

Comment:
We recently introduced a method to set the two different cookies, so this will be:

```suggestion
    this.setVerifierCookie(verifier);
```